examples/covasim_/vaccinating_elderly/example_vaccine.py

```diff
@@ -1,38 +1,34 @@
 # -*- coding: utf-8 -*-
+import os
+import logging
+import warnings
 import pandas as pd
-import numpy as np
-import covasim as cv  # Version used in our study is 3.07
-import random
 from causal_testing.specification.causal_dag import CausalDAG
 from causal_testing.specification.scenario import Scenario
 from causal_testing.specification.variable import Input, Output
 from causal_testing.specification.causal_specification import CausalSpecification
-from causal_testing.data_collection.data_collector import ExperimentalDataCollector
+from causal_testing.data_collection.data_collector import ObservationalDataCollector
 from causal_testing.testing.causal_test_case import CausalTestCase
 from causal_testing.testing.causal_test_outcome import Positive, Negative, NoEffect
 from causal_testing.testing.estimators import LinearRegressionEstimator
 from causal_testing.testing.base_test_case import BaseTestCase
 
-import os
-import logging
+warnings.filterwarnings("ignore", message=".*The 'nopython' keyword.*")  # Supress the numba warning in covasim
 
 logger = logging.getLogger(__name__)
 logging.basicConfig(level=logging.DEBUG, format="%(message)s")
-
 ROOT = os.path.realpath(os.path.dirname(__file__))
 
 
-def test_experimental_vaccinate_elderly(runs_per_test_per_config: int = 30, verbose: bool = False):
-    """Run the causal test case for the effect of changing vaccine to prioritise elderly. This uses the experimental
-        data collector.
+def setup_test_case(verbose: bool = False):
+    """Run the causal test case for the effect of changing vaccine to prioritise elderly from an observational
+    data collector that was previously simulated.
 
-    :param runs_per_test_per_config: Number of times to run each input configuration (control and treatment) per test.
-                                     Hence, the total number of runs per test will be twice this value.
     :param verbose: Whether to print verbose details (causal test results).
     :return results_dict: A dictionary containing ATE, 95% CIs, and Test Pass/Fail
     """
 
-    # 1. Read in the Causal DAG
+    # 1. Create causal dag
```

Comment:
To me the previous comment was more clear, is there a reason for this change?

Reply:
There isn't at all - no idea why I changed the original comment!